web-common/src/components/data-graphic/marks/Line.svelte

```diff
@@ -62,7 +62,6 @@
       yScale: $yScale,
       curve,
       xAccessor,
-      pathDefined,
```

Comment:
note: no need to pass `pathDefined = pathDoesNotDropToZero`, because `pathDoesNotDropToZero` is the dafault when this argument is omitted